iwyu_test_util.py

```diff
@@ -69,7 +69,8 @@ def _Which(program, paths):
 
 def _GetIwyuPath(iwyu_paths):
   """Returns the path to IWYU or raises IOError if it cannot be found."""
-  iwyu_path = _Which('include-what-you-use', iwyu_paths)
+  #iwyu_path = _Which('include-what-you-use', iwyu_paths)
+  iwyu_path = "/home/chris/src/build/llvm/_target/gcc/bin/include-what-you-use"
```

Comment:
Oops, we probably don't want to merge this :-)

The test runner is designed to search a number of known paths + the system `PATH`. So you can add your IWYU build dir to the `PATH` in your shell and it will find it.